packages/adblocker/src/filters/network.ts

```diff
@@ -1368,6 +1368,14 @@ export default class NetworkFilter implements IFilter {
     return this.optionValue ?? '';
   }
 
+  public getRedirectResource(): string {
+    return this.getRedirect().split(':')[0];
+  }
+
+  public getRedirectPriority(): number {
```

Comment:
Don't know if it is used in a performance critical path. If so, it could be optimized.

Reply:
I created a branch to handle this: https://github.com/ghostery/adblocker/pull/4219

Reply:
it's not a critical path. those are called after filters are already matched